Dangerfile

```diff
@@ -20,14 +20,33 @@ end
 is_adding_draft = !(git.added_files.grep(/_drafts/).empty?)
 is_editing_draft = !(git.modified_files.grep(/_drafts/).empty?)
 if is_editing_draft || is_adding_draft
-  warn("Looks like you are editing a draft. Don't forget to publish when it's finished!")
+    warn("Looks like you are editing a draft. Don't forget to publish when it's finished!")
 end
 
 # -----------------------------------------------------------------------------
 # Check spelling and prose
 # -----------------------------------------------------------------------------
 prose.language = "en-us"
+prose.ignore_numbers = true
+prose.ignore_acronyms = true
 prose.lint_files
 
 prose.ignored_words = ["twc", "TWC"]
 prose.check_spelling
+
+# -----------------------------------------------------------------------------
+# Verify Jekyll template
+#
+# Borrowed from: https://github.com/artsy/artsy.github.io/blob/master/Dangerfile
+# -----------------------------------------------------------------------------
+active_files = (git.modified_files + git.added_files).uniq
+markdowns = active_files
```

Comment:
Extremely minor point, but it would be more accurate to name this variable `posts`, instead of `markdowns`, since it won't include e.g. `about.md` (which I believe is exactly what you intend, since you are checking for the presence of an `excerpt` break).

Reply:
ah. yes. will rename. 💯